src/test/ui/fat-ptr-cast.rs

```diff
@@ -19,6 +19,12 @@ fn main() {
     q as *const [i32]; //~ ERROR cannot cast
 
     // #21397
+
     let t: *mut (dyn Trait + 'static) = 0 as *mut _; //~ ERROR casting
     let mut fail: *const str = 0 as *const str; //~ ERROR casting
+
+    let x: *mut (dyn Trait + 'static) = std::ptr::null_mut();
+    //~^ ERROR E0277
```

Comment:
cc #61870

Reply:
Might be best to leave this one unchanged if it breaks the test.

Reply:
Okey thanks! I removed it.